src/format.iced

```diff
@@ -0,0 +1,114 @@
+# This class prettifies the output of our armoring stream, according to the saltpack spec.
+# At the moment, per the spec, it simply frames the message, and inserts a space every 15 characters
+# and a newline every 200 words.
+
+stream = require('keybase-chunk-stream')
+util = require('./util')
+
+# Punctuation - this is modular
+space = new Buffer(' ')
+newline = new Buffer('\n')
+punctuation = new Buffer('.')
+
+words_per_line = 200
+chars_per_word = 15
+
+# For working with the chunk streams
+noop = () ->
+
+# This stream takes basex'd input and writes framed output, as per https://saltpack.org/armoring#framing-the-basex-payload
+exports.FormatStream = class FormatStream extends stream.ChunkStream
+
+  # _format is the transform function passed to the chunk stream constructor
+  _format : (chunk) ->
+    # this is only ever called during flush()
+    if chunk.length < @block_size
+      return chunk
```

Comment:
We're relying on this short-circuit to avoid printing an extra space/newline after the very last block. But if the last block happens to be exactly equal to the output block size, we'll fail to short-circuit, and an incorrect space/newline will go in. We should probably insert these characters _before_ each block, and add some logic to skip doing that for the first block.